ctapipe/ml/tools/tests/test_train.py

```diff
@@ -94,3 +120,26 @@ def test_cross_validation_results(model_tmp_path):
     )
     assert ret == 0
     assert classifier_cv_out_file.exists()
+
+    tool = TrainDispReconstructor()
+    out_file_reg = model_tmp_path / "disp_regressor.pkl"
+    out_file_clf = model_tmp_path / "sign_classifier.pkl"
+    disp_cv_out_file_base = model_tmp_path / "disp_cv_results.h5"
+
+    disp_reg_cv_out_file = model_tmp_path / "disp_cv_results_regressor.h5"
+    disp_clf_cv_out_file = model_tmp_path / "disp_cv_results_classifier.h5"
+
+    ret = run_tool(
+        tool,
+        argv=[
+            "--input=dataset://gamma_diffuse_dl2_train_small.dl2.h5",
+            f"--output-regressor={out_file_reg}",
+            f"--output-classifier={out_file_clf}",
+            f"--config={config}",
+            "--log-level=INFO",
+            f"--CrossValidator.output_path={disp_cv_out_file_base}",
+        ],
+    )
+    assert ret == 0
+    assert disp_reg_cv_out_file.exists()
+    assert disp_clf_cv_out_file.exists()
```

Comment:
This feels off, I cannot articulate why.

Reply:
Do you mean the naming (maybe use `disp_cv_out_file_{reg,clf}` instead)? Or do you mean this will/ should not work correctly?

Reply:
I changed the naming as stated above. Marking this as resolved for now.